rs-matter-macros/Cargo.toml

```diff
@@ -22,7 +22,7 @@ proc-macro = true
 syn = { version = "2.0.48", features = ["extra-traits", "parsing"] }
 quote = "1"
 proc-macro2 = "1"
-proc-macro-crate = "3.0.0"
+proc-macro-crate = "3.1.0"
```

Comment:
your description of the PR said this should be "3" instead of 3.1.0 and same with syn. Should we do that instead?

Reply:
That would be the better solution.